<?xml version="1.0" encoding="UTF-8"?><record_update table="syntax_editor_macro">
    <syntax_editor_macro action="INSERT_OR_UPDATE">
        <comments>Client-side try/catch </comments>
        <name>Try</name>
        <sys_class_name>syntax_editor_macro</sys_class_name>
        <sys_created_by>admin</sys_created_by>
        <sys_created_on>2020-03-12 12:34:43</sys_created_on>
        <sys_id>ecf673bc2f6340108442d4a72799b6fd</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>Try</sys_name>
        <sys_package display_value="NeedIt" source="x_58872_needit">6ead8e780f603200cd674f8ce1050ed1</sys_package>
        <sys_policy/>
        <sys_scope display_value="NeedIt">6ead8e780f603200cd674f8ce1050ed1</sys_scope>
        <sys_update_name>syntax_editor_macro_ecf673bc2f6340108442d4a72799b6fd</sys_update_name>
        <sys_updated_by>admin</sys_updated_by>
        <sys_updated_on>2020-03-12 12:34:43</sys_updated_on>
        <text>try{&#13;
&#13;
}&#13;
catch(err){&#13;
g_form.addErroMessage("A runtime error occured:" + err);&#13;
}</text>
    </syntax_editor_macro>
</record_update>
